addons/explore-editor-theme/ui/IconSaver.gd

```diff
@@ -0,0 +1,34 @@
+tool
+extends VBoxContainer
+
+
+signal filesystem_changed()
+
+export var icon_name := ''
```

Comment:
Here and below there is also no need for single quotes. Also consider adding explicit types here.